llama_stack/providers/inline/vector_io/sqlite_vec/sqlite_vec.py

```diff
@@ -27,7 +27,12 @@
 # Specifying search mode is dependent on the VectorIO provider.
 VECTOR_SEARCH = "vector"
 KEYWORD_SEARCH = "keyword"
-SEARCH_MODES = {VECTOR_SEARCH, KEYWORD_SEARCH}
+HYBRID_SEARCH = "hybrid"
+SEARCH_MODES = {VECTOR_SEARCH, KEYWORD_SEARCH, HYBRID_SEARCH}
```

Comment:
we discussed moving to search strategy now but, as you and I discussed, it's better to introduce all of those breaking changes at once rather than in pieces, so keeping this makes sense.